data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea

```diff
@@ -0,0 +1,2132 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<gnc-account-example
+     xmlns:gnc="http://www.gnucash.org/XML/gnc"
+     xmlns:act="http://www.gnucash.org/XML/act"
+     xmlns:book="http://www.gnucash.org/XML/book"
+     xmlns:cd="http://www.gnucash.org/XML/cd"
+     xmlns:cmdty="http://www.gnucash.org/XML/cmdty"
+     xmlns:price="http://www.gnucash.org/XML/price"
+     xmlns:slot="http://www.gnucash.org/XML/slot"
+     xmlns:split="http://www.gnucash.org/XML/split"
+     xmlns:sx="http://www.gnucash.org/XML/sx"
+     xmlns:trn="http://www.gnucash.org/XML/trn"
+     xmlns:ts="http://www.gnucash.org/XML/ts"
+     xmlns:fs="http://www.gnucash.org/XML/fs"
+     xmlns:bgt="http://www.gnucash.org/XML/bgt"
+     xmlns:recurrence="http://www.gnucash.org/XML/recurrence"
+     xmlns:lot="http://www.gnucash.org/XML/lot"
+     xmlns:addr="http://www.gnucash.org/XML/addr"
+     xmlns:billterm="http://www.gnucash.org/XML/billterm"
+     xmlns:bt-days="http://www.gnucash.org/XML/bt-days"
+     xmlns:bt-prox="http://www.gnucash.org/XML/bt-prox"
+     xmlns:cust="http://www.gnucash.org/XML/cust"
+     xmlns:employee="http://www.gnucash.org/XML/employee"
+     xmlns:entry="http://www.gnucash.org/XML/entry"
+     xmlns:invoice="http://www.gnucash.org/XML/invoice"
+     xmlns:job="http://www.gnucash.org/XML/job"
+     xmlns:order="http://www.gnucash.org/XML/order"
+     xmlns:owner="http://www.gnucash.org/XML/owner"
+     xmlns:taxtable="http://www.gnucash.org/XML/taxtable"
+     xmlns:tte="http://www.gnucash.org/XML/tte"
+     xmlns:vendor="http://www.gnucash.org/XML/vendor">
+  <gnc-act:title>
+      Plan comptable PME CH 2019
+  </gnc-act:title>
+  <gnc-act:short-description>
+     Nouveau modèle 2019 plan comptable PME Suisse  
+  </gnc-act:short-description>
+  <gnc-act:long-description>
+     Plan comptable PME selon Sterchi I Mattle I Helbling publié par veb.ch
+  </gnc-act:long-description>
+  <gnc-act:exclude-from-select-all>1</gnc-act:exclude-from-select-all>
+  <gnc:account version="2.0.0">
+    <act:name>Root Account</act:name>
+    <act:id type="guid">8cd23636f1e04a74a35c13ae23427c14</act:id>
+    <act:type>ROOT</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Actifs</act:name>
+    <act:id type="guid">e3f7b59890ec44688241f322db0a7392</act:id>
+    <act:type>ASSET</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>1</act:code>
+    <act:description>1</act:description>
```

Comment:
```
  <act:slots>
    <slot>
      <slot:key>placeholder</slot:key>
      <slot:value type="string">true</slot:value>
    </slot>
  </act:slots>
```
is missing here and later in all accounts,which are no leaves of the account tree.